javascript/src/seedsource-ui/containers/TransferStep.tsx

```diff
@@ -28,6 +28,42 @@ type TransferStepProps = ConnectedProps<typeof connector> & {
   active: boolean
 }
 
+const helpTooltip = (
+  <>
+    <p>
+      {t`The transfer limit method selected determines how far climatically a seedlot may be moved and still 
+      have an acceptable level of adaptation. `}
+    </p>
+    <br />
+    <p>{t`The SST uses four approaches to determine and map transfer limits: `}</p>
```

Comment:
The spacing feels a little excessive here. Try dropping the `<br />` and adjusting paragraph spacing with CSS.